examples/react-app-ts/jest.setup.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+require("@testing-library/jest-dom");
```

Comment:
this is a great PR thanks!

though I would also love to use the module format here:
```
import '@testing-library/jest-dom';
```
like the other react example https://github.com/sodatea/vite-jest/blob/main/examples/react-app-type-module/src/setupTests.js